events-application-rest-api/pom.xml

```diff
@@ -54,7 +54,6 @@
         <dependency>
             <groupId>org.postgresql</groupId>
             <artifactId>postgresql</artifactId>
-            <version>9.4-1203-jdbc42</version>
```

Comment:
I am using another version that's why I have removed it. However, I have added it again it's better :smile: